### PR TITLE
Sprint prep

### PR DESCRIPTION
**Pull request by b5**
Sprint prep checklist. This PR should:
* Update repo to new package names (`archive` -> `core`, etc.)
* Add stub `README.md`, moving old readme to readme_prev.md if it existed
* Add `.github` stuff: CONTRIBUTING, ISSUES
* Add support for codecov.io to CI tests
* confirm existence of a `LICENSE` file

### CODE REVIEW

File: readme.md
Comment (dcwalk) [inline readme.md]: Looks like the wrong readme got copied in again

**Review by dcwalk — CHANGES_REQUESTED** [review]
readme issue but otherwire 👍

**Comment by dcwalk** [comment]
This PR should address #18  and https://github.com/datatogether/roadmap/issues/53

**Review by dcwalk — APPROVED** [review]

**Comment by b5** [comment]
Should we also add codecov badges?

**Comment by dcwalk** [comment]
@b5 --  your call.

**Comment by b5** [comment]
@patcon here's a working example, it should switch from "unknown" once we merge to master. is this ready to merge @dcwalk?

**Review by titaniumbones — APPROVED** [review]
Looks good to me. Is there any reason not to merge @b5?

**Comment by patcon** [comment]
Ok, two +1 makes this good to go!

**Pull request merged**
